src/olympia/devhub/templates/devhub/addons/dashboard.html

```diff
@@ -45,7 +45,7 @@ <h3>
             {{ item }}
             <span class="activity-timestamp">
             {% trans user=item.user|user_link, ago=item.created|timesince %}
```

Comment:
It's this line you'll want to change, where `user` is set to the result of the `user_link` jinja filter.  To just have the name but not the link we can just use `user=user.name` instead.